diskquota.c

```diff
@@ -530,6 +531,12 @@ disk_quota_worker_main(Datum main_arg)
 			/* Refresh quota model with init mode */
 			refresh_disk_quota_model(!MyWorkerInfo->dbEntry->inited);
 			MyWorkerInfo->dbEntry->inited = true;
+			is_gang_destroyed             = false;
+		}
+		else if (!is_gang_destroyed)
```

Comment:
```suggestion
		if (!is_gang_destroyed)
```
Can we change `else if` to `if`? Because it should be triggered in the next loop if `else if` is used.

Reply:
> Can we change else if to if? Because it should be triggered in the next loop if else if is used.
No, if the diskquota is not paused, we would like to reuse the gang.

Reply:
Why we should destroy all gangs when `diskquota is not paused` and `gang is not destroyed`?

Reply:
> Why we should destroy all gangs when diskquota is not paused and gang is not destroyed?

We destroy gangs when `diskquota is paused and gang is not destoried`

The user can pause diskquota and resume it, when diskquota is paused, the diskquota worker process still runs, but does nothing, so we just destroy the gang.  And if the gang is closed, there is no need to close it again.

Reply:
OK, I have mistaken it, now it is clear.